Author: Holger Jaekel <holger.jaekel@gmx.de>
Summary: @achrinza/node-ipc@9.2.9 explicitly restricts the allowed Node.js versions to <= 22
----

--- a/resources/CMakeLists.txt
+++ b/resources/CMakeLists.txt
@@ -47,7 +47,7 @@
   # add_custom_command does not support OUTPUT_QUIET, so we need to make a cmake subcommand first:
   file(WRITE ${CMAKE_BINARY_DIR}/yarn_commands.cmake
   "execute_process(
-      COMMAND ${YARN} install --frozen-lockfile
+      COMMAND ${YARN} install --frozen-lockfile --ignore-engines
       WORKING_DIRECTORY ${LANDINGPAGE_OUTPUT_PATH}
       OUTPUT_QUIET
       ERROR_FILE ${CMAKE_BINARY_DIR}/yarn_error.log
